.gitignore

```diff
@@ -26,4 +26,7 @@ nuget.exe
 *.ncrunchsolution
 *.*sdf
 *.ipch
-*.sln.ide
\ No newline at end of file
+.settings
```

Comment:
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: 

Some people might object but good god the settings files have driven me mad the past week.